README.md

```diff
@@ -611,6 +611,85 @@ xy = foo(x, y=3)
     """
     ```
 
+- Group similar one line statements together.
+
+    ```julia
+    # Yes
+    foo = 1
+    bar = 2
+    baz = 3
+
+    # No
+    foo = 1
+
+    bar = 2
+
+    baz = 3
+    ```
+
+- Use blank-lines to separate different multi-line blocks.
+
+    ```julia
+    # Yes
+    if foo
+      println("Hi")
+    end
+
+    for i in 1:10
+      println(i)
+    end
+
+    # No
+    if foo
+        println("Hi")
+    end
+    for i in 1:10
+        println(i)
+    end
+    ```
+
+- After a function definition, and before an end statement do not include a line break
+
+    ```julia
+    # Yes
+    function foo(bar::Int64, baz::Int64)
+        return bar + baz
+    end
+
+    # No
+    function foo(bar::Int64, baz::Int64)
+
+        return bar + baz
+    end
+
+    # No
+    function foo(bar::In64, baz::Int64)
+        return bar + baz
+
+    end
+    ```
+
+- Use line breaks between control flow statements and returns
```

Comment:
```suggestion
- Use line breaks between control flow statements and returns.
```

Reply:
This is supposed to be optional? Can we specifiy that?

Reply:
From #54 this is not optional. It's only optional for single line statements preceding it, e.g.

```julia
# Optional
function foo(bar)
    bar = bar + 1
    baz = bar * 3

    return (bar, baz)
end

# Acceptable
function foo(bar)
    bar = bar + 1
    baz = bar * 3
    return (bar, baz)
end
```